cob_moveit_config/package.xml

```diff
@@ -13,5 +13,6 @@
   <buildtool_depend>catkin</buildtool_depend>
 
   <exec_depend>xacro</exec_depend>
+  <exec_depend>trac_ik</exec_depend>	
```

Comment:
sorry, I guess the convention is to keep dependencies of `cob_moveit_config` simple...but add them to `cob_moveit_bringup`...I did this in https://github.com/ipa320/cob_manipulation/pull/116...so please remove the dependency here...sorry